webcompat/static/js/lib/bugform-validation.js

```diff
@@ -24,6 +24,10 @@ function Validation() {
       var val = descField.val();
       return $.trim(val) !== "";
     },
+    isIssueValid: function(issueField) {
```

Comment:
This seems like it's doing the same as `isDescriptionValid` -- in the future we might consider making them call the same method. But let's merge here to not block.

Reply:
https://github.com/webcompat/webcompat.com/issues/2992